### DISCUSSION TITLE
Hide results from Hyperlink field if they are empty

### DISCUSSION THREAD

**Discussion by Tunni283** [microsoft-search/pnp-modern-search#4055] [Q&A] [answered]
Version used: 4.13.1.0

Question
I am using a Hyperlink field in my list (site column) and have the following handlebar expression in the manage columns section to render the URL title.

`<a href="{{slot item @root.slots.HRLink}}" target="_blank" style="color:{{@root.theme.semanticColors.link}}" data-interception="off">{{getUrlField HRLinkOWSURLH "Title"}}</a>`

The problem I have is that if there is no value in the column for a list item, it renders the columns with null in the hyperlink field.  Is there anything I can do to hide these null values and only show the populated hyperlinks?

**Comment by kasperbolarsen** [comment, marked as answer]
I guess it should be possible to wrap it in an If clause?
 {{#if (slot item @root.slots.HRLink)}}
<a href="{{slot item @root.slots.HRLink}}" target="_blank" style="color:{{@root.theme.semanticColors.link}}" data-interception="off">{{getUrlField HRLinkOWSURLH "Title"}}</a>
{{/if}}

**Reply by Tunni283** [reply]
Thank you that did the trick!
